sound/soc/sof/core.c

```diff
@@ -104,6 +104,8 @@ static int sof_probe_continue(struct snd_sof_dev *sdev)
 		return ret;
 	}
 
```

Comment:
@ranj063 if it is not a big trouble I would like to see a state diagram in the commit message. With possible states and transitions. This will help review and understand the code easier.

Reply:
+1

Reply:
@plbossart @dbaluta done